html/captive-portal/templates/layout.html

```diff
@@ -1,12 +1,14 @@
 <html>
   <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <link href="/content/new-styles.css" media="screen" rel="stylesheet" type="text/css" />
     <script src="/common/jquery-1.11.3.min.js" type="text/javascript"></script>
     <script src="/common/pf.js" type="text/javascript"></script>
+    <title>[% IF title.defined ; i18n(title) ; ELSE ; i18n("PacketFence Registration system") ; END %]</title>
   </head>
   <h1>PacketFence captive portal</h1>
   [% FOREACH level IN flash.keys %]
```

Comment:
You are right.

Not sure if this is really an issue though.

Not sure either what it has to do with this PR